libbpfgo/helpers/argParsers_test.go

```diff
@@ -0,0 +1,15 @@
+package helpers
+
+import (
+	"fmt"
+	"os"
+)
+
+func ExampleTracePipeListen_usage() {
```

Comment:
This file is named as `argParsers_test.go` but none of the functions are actually test functions themselves. If it's more of an example/toy program to demonstrate usage of `TracePipe()`, I would suggest renaming the file. 

Otherwise we should make `TracePipeListen()` injectable with a param, something like: `TracePipeListen(path string)`, path being something we can inject during a test. We can further use `"/sys/kernel/debug/tracing/trace_pipe"` as the default if no args are specified (the real world usage) to alleviate the need for passing in the path for the library user.

Reply:
Good idea on making that function testable, I'd like to go through a lot of these helpers and write tests, will do I that in a follow up PR (I swear, I will!) 

I've gone ahead and renamed this file to `example_tracelisten_test.go`. The `_test` suffix is necessary for the example to show up in godoc, but it seems an `example_` prefix is idiomatic in the standard library (for example: [code](https://github.com/golang/sys/blob/c4fcb01b228e8541206c34b2b968566305616517/unix/example_exec_test.go) and [docs](https://pkg.go.dev/golang.org/x/sys/unix#example-Exec)).

Reply:
the tests in `tracee-ebpf/tracee/tracee_test.go` are mostly testing these argprinters. I think we can move them here (argparsers_test.go)

also, `TracePipeListen` doesn't feel like it belongs in the argParsers file, can we move it into another `helpers` file? if we do that, I think it's ok to rename `example_tracelisten_test.go` to `helpers_test.go` (it's common in go to put examples with test right?). wdyt?

Reply:
@itaysk I don't follow, the only test I see in `tracee_test.go` is for `readArgFromBuff()`. 

I am all for separating out TracePipeListen to its own file and renaming the test file though. I'm about to open a PR with helpers for CO:RE stuff, I'll do that change as part of that PR.